src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -70,21 +65,17 @@ const defaults: IDropdownOptions = {
 	strictSelection: true,
 	maxHeight: 300,
 	errorMessage: errorMessage,
-	contextBorder: Color.fromHex('#696969'),
-	actionLabel: nls.localize('dropdownAction.toggle', "Toggle dropdown")
+	contextBorder: Color.fromHex('#696969')
```

Comment:
It would be great to use the color registry for this instead.

Reply:
this is a fallback color, and we don't actually need this, I am removing it.